doc/install/ARCHLINUX.md

```diff
@@ -1,37 +1,34 @@
-Install IKOS on Archlinux
-=========================
+Install IKOS on Arch Linux
+==========================
 
-Here are the steps to install IKOS on **[Archlinux](https://www.archlinux.org/)** using **[AUR](https://aur.archlinux.org/)** (Archlinux User Repositories).
+IKOS can be installed on **[Arch Linux](https://www.archlinux.org/)** by building and installing the *apron* and *ikos* **[AUR](https://aur.archlinux.org/)** packages. See **[Installing packages](https://wiki.archlinux.org/index.php/Arch_User_Repository#Installing_packages)** for detailed instructions.
 
-First, make sure your system is up-to-date:
-
-```
-$ sudo pacman -Syu
-```
-
-Now, install IKOS using your favorite **AUR** helper:
-
-```
-$ yay -S ikos
-```
+* https://aur.archlinux.org/packages/apron
+* https://aur.archlinux.org/packages/ikos
 
 You are now ready to use IKOS. Go to the section [How to Run IKOS](../../README.md#how-to-run-ikos) in README.md
 
-Build IKOS from source on Archlinux
-===================================
+To be informed on updates by email, click on "Enable notifications" in "Package actions" on the AUR website.
 
-Here are the steps to install the required dependencies to build IKOS from source on **[Archlinux](https://www.archlinux.org/)** using **[AUR](https://aur.archlinux.org/)** (Archlinux User Repositories).
+Build IKOS from source on Arch Linux
+====================================
+
+Here are the steps to install the required dependencies to build IKOS from source on **[Arch Linux](https://www.archlinux.org/)** using **[AUR](https://aur.archlinux.org/)** (Arch User Repository).
 
 First, make sure your system is up-to-date:
 
 ```
 $ sudo pacman -Syu
 ```
 
-Now, install the following packages using your favorite **AUR** helper:
+Now, install the following packages using **pacman**:
 
 ```
-$ yay -S base-devel cmake gmp boost boost-libs python python-pygments sqlite llvm llvm-libs clang apron-ppl-svn
+$ pacman -S base-devel cmake gmp boost boost-libs python python-pygments sqlite llvm llvm-libs clang
 ```
 
+Now install and build the apron-ppl-svn package.
+
+* https://aur.archlinux.org/packages/apron-ppl-svn
```

Comment:
Please rename 'apron-ppl-svn' in 'apron'. I removed the package 'apron-ppl-svn' yesterday. Same thing 2 lines above.

Reply:
Done